docker/prognostic_run/Dockerfile

```diff
@@ -78,13 +84,9 @@ RUN cd /tmp/fortran-build/FV3 && \
     make -j 8 && \
     PREFIX=/usr/local make install
 
-# gcloud
+COPY docker/prognostic_run/requirements.txt /tmp/requirements.txt
 
-#Install gcloud
-RUN apt-get update && apt-get install -y  apt-transport-https ca-certificates gnupg curl gettext && \
-    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list &&\
-    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add - && \
-    apt-get update && apt-get install -y google-cloud-sdk jq python3-dev python3-pip kubectl gfortran graphviz
+RUN pip3 install -c /tmp/requirements.txt numpy cython jinja2 && make -C /tmp/fortran-build/FV3 wrapper_build
```

Comment:
Do we need jinja, if we aren't building?

Should numpy and cython go in docker/prognostic_run/requirements.txt instead?

Reply:
we are building it.

numpy and cython are in the requiremnts file, but I wanted to isolate the wrapper building to an earlier part of the makefile to allow better caching.